### DISCUSSION TITLE
TIP de HTML

### DISCUSSION THREAD

**Discussion by SandroBorrallez** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#144] [Ideas]
Espero y les sirva este TIP para los que aun estamos iniciando con HTML, para evitar la captura del código inicial, pueden hacerlo casi de forma automática, solamente con lo siguiente:

En la primer línea de código en HTML, solo pongan el signo de exclamación (!) sin paréntesis y presionan enter, y listo!!

Espero y les sea de ayuda para los que aún no lo sabían

**Comment by Ingrzs** [comment]
Añado si no les sale solo empiecen a tipear html y les saldra sugerencias eligan html o html5, yo elegi este último y igual todo se genera automáticamente. 
(Claro esta que deben ya tener guardado el archivo como html)

**Comment by khisus19** [comment]
En VScode si no te aparecen las sugerencias de autocompletado presiona Ctrl + BarraEspaciadora
